$NetBSD: patch-af,v 1.1.1.1 2006/07/25 16:28:47 salo Exp $

add necessary header file, and function prototype to kill compiler warnings

--- parser/cmtp.c.orig	2003-09-13 00:38:11.000000000 +0100
+++ parser/cmtp.c
@@ -30,10 +30,13 @@
  */
 
 #include <sys/types.h>
+#include <sys/time.h>
 #include <stdio.h>
 
 #include "parser.h"
 
+char *bst2str(uint8_t);
+
 char *bst2str(uint8_t bst)
 {
 	switch (bst) {
